terraform/azure_rke2_cluster/examples/gmsa.tfvars

```diff
@@ -6,11 +6,10 @@ nodes = [
     replicas = 1
   },
   {
-    name        = "windows-server"
-    image       = "windows"
-    roles       = ["worker"]
-    replicas    = 1
-    domain_join = true
```

Comment:
default is now changed to non domain joined, the `gmsa_dj.tfvars` is now the one that tests domain joining.